public-api/src/main/java/org/pmiops/workbench/cdr/dao/AchillesAnalysisDao.java

```diff
@@ -1,9 +1,23 @@
 package org.pmiops.workbench.cdr.dao;
 import org.pmiops.workbench.cdr.model.AchillesAnalysis;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.List;
 
 public interface AchillesAnalysisDao extends CrudRepository<AchillesAnalysis, Long> {
     List<AchillesAnalysis> findAll();
+
+    @Query(value = "select distinct a from AchillesAnalysis a left join FETCH a.results as r " +
+            "where r.stratum1 = ?1 and r.stratum2 = ?2 order by a.analysisId"
+    )
+    List<AchillesAnalysis> findQuestionAnalysisResults(String survey_concept_id, String question_concept_id);
+
+    @Query(value = "select distinct a from AchillesAnalysis a left join FETCH a.results as r " +
+            "where r.stratum1 = ?1 and r.stratum2 in (?2) order by a.analysisId"
+    )
+    List<AchillesAnalysis> findSurveyAnalysisResults(String survey_concept_id, List<String> question_concept_ids);
```

Comment:
Why do we need both of these methods? (Are the analyses returned from this method enough that the client doesn't have to ask for analyses later on a per-question basis?)

Reply:
Good point. we really just need to be able to pass a list of questions for search results and whole surveys.  Removed.